app/controllers/applications_controller.rb

```diff
@@ -4,4 +4,23 @@ def show
     @application = Application.find(params[:id])
     @pets = @application.pets
   end
+
+  def new
+  end
+
+  def create
+    application = Application.new({
```

Comment:
Like you mentioned in your PR overview, we could refactor this to .create to do both new and save simultaneously but not all that crucial. We can make those tweaks toward the end if need be.